compile/src/test/scala/sbt/compiler/javac/JavaCompilerSpec.scala

```diff
@@ -64,20 +65,28 @@ object JavaCompilerSpec extends Specification {
     errored and foundErrorAndWarning and hasKnownErrors
   }
 
-  def lineMatches(p: Problem, lineno: Int): Boolean =
-    p.position.line.isDefined && (p.position.line.get == lineno)
+  def lineMatches(p: Problem, lineno: Int, lineContent: Option[String] = None): Boolean = {
+    def lineContentCheck =
+      lineContent match {
+        case Some(content) => content.equalsIgnoreCase(p.position.lineContent())
+        case _             => true
+      }
+    def lineNumberCheck = p.position.line.isDefined && (p.position.line.get == lineno)
```

Comment:
Option.get should be avoided:

    def lineNumberCheck = p.position.line.exists(_ == lineno)